pages.de/common/pip.md

```diff
@@ -0,0 +1,25 @@
+# pip
+
+> Python package manager.
+> Einige Unterbefehle wie `pip install` sind separat dokumentiert.
+> Weitere Informationen: <https://pip.pypa.io>.
+
+- Installiere ein Paket (siehe `pip install` für weitere Beispiele):
+
+`pip install {{paketname}}`
+
+- Upgrade ein Paket:
```

Comment:
What do you think about `aktualisiere` here?

Reply:
Good idea, I'll change it! Thank you